README.md

```diff
@@ -49,6 +49,30 @@ redis:
 
 Consult the [Icinga DB configuration documentation] on what options there are.
 
+### Connect via TLS
+
+```bash
+docker run -d \
+	--network icinga \
+	--restart always \
+	-e ICINGADB_REDIS_ADDRESS=redis-icingadb:6379 \
```

Comment:
Outdated configuration variable. Please look at #49 and then also change it in this PR.